frontend/appflowy_flutter/lib/plugins/document/presentation/layout/sizes.dart

```diff
@@ -0,0 +1,6 @@
+import 'package:flutter/material.dart';
+
+class DocumentSizes {
+  static const double menuHeight = 200.0;
```

Comment:
What do these values represent? It appears that they are only used in the emoji event.

Reply:
It was suggested to move those values to some kind of class since they might be used in the future